avocado/utils/process.py

```diff
@@ -274,8 +274,8 @@ class CmdResult(object):
     :param pid: ID of the process
     :type pid: int
     :param encoding: the encoding to use for the text version
-                     of stdout and stderr, with the default being
-                     Python's own (:func:`sys.getdefaultencoding`).
+                     of stdout and stderr, by default
+                     :mod:`avocado.utils.astring.ENCODING`
```

Comment:
Please use

```
:data:`avocado.utils.astring.ENCODING`
```

Instead.  According to http://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#directive-py:data

Reply:
ok